config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -278,6 +200,42 @@ periodics:
           cpu: 6
           memory: "16Gi"
 
+- cron: '1 8 * * 6' # Run at 00:01PST (8:01 UTC) on Sat
+  name: ci-kubernetes-e2e-gke-scale-performance
+  tags:
+  - "perfDashPrefix: gke-5000Nodes"
+  - "perfDashJobType: performance"
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+  spec:
+    containers:
+    - args:
+      - --timeout=1320
+      - --bare
+      - --scenario=kubernetes_e2e
+      - --
+      - --cluster=gke-scale-cluster
+      - --deployment=gke
+      # TODO(wojtek-t): get rid of this once GKE is fixed in master
+      - --extract=ci/latest-1.13
+      - --gcp-cloud-sdk=gs://cloud-sdk-testing/ci/staging
+      - --gcp-node-image=gci
+      - --gcp-project=kubernetes-scale
+      - --gcp-zone=us-east1-a
+      - --gke-create-command=container clusters create --quiet --enable-ip-alias --create-subnetwork name=ip-alias-subnet --cluster-ipv4-cidr=/11 --services-ipv4-cidr=/18 --timeout=2700
+      - --gke-environment=staging
+      - '--gke-shape={"default":{"Nodes":4999,"MachineType":"n1-standard-1"},"heapster-pool":{"Nodes":1,"MachineType":"n1-standard-16"}}'
+      - --provider=gke
+      - --test_args=--ginkgo.focus=\[Feature:Performance\]
+      - --timeout=1290m
+      - --use-logexporter
+      image: gcr.io/k8s-testimages/kubekins-e2e:v20190117-9a712c1e6-master
+      resources:
+        requests:
+          cpu: 6
+          memory: "16Gi"
+
```

Comment:
Separately, can you please update this document:
https://github.com/kubernetes/community/blob/master/sig-scalability/processes/scalability-validation.md
?

Reply:
And the calendar linked in that doc :D

Reply:
Yeah - that's mostly what I meant :-)

Reply:
Done - https://github.com/kubernetes/community/pull/3122